plugins/csv_logs.yaml

```diff
@@ -0,0 +1,64 @@
+version: 0.0.1
+title: CSV
+description: Log parser for CSV
+parameters:
+  - name: log_paths
+    type: "[]string"
+    required: true
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+  - name: exclude_log_paths
+    type: "[]string"
+    default: []
+  - name: header
+    type: string
+    required: true
+  - name: encoding
+    type: string
+    supported:
+      - nop
+      - utf-8
+      - utf-16le
+      - utf-16be
+      - ascii
+      - big5
+    default: utf-8
+  - name: log_type
+    type: string
+    default: csv
+  - name: start_at
+    type: string
+    supported:
+    - beginning
+    - end
+    default: end
```

Comment:
`start_at` is in here twice. It looks like it is in the original too.

Reply:
I just removed one of the repeated start-at. I redid some testing and it appear to still work just fine.

Reply:
Also, I did test against some log files, it seemed to function correctly.